### PR TITLE
[Master] - Fix configuring identity analytics Doc Issues

### PR DESCRIPTION
**Pull request by Sachin-Mamoru**
## Automation tests
 - Unit tests 
   > N/A
 - Integration tests
   > N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/docs/learn/configuring-identity-analytics.md [hunk 1]
```diff
@@ -6,27 +6,60 @@ Using the WSO2 Identity Server Analytics distribution, you can view and analyze
...
 
-To do this, the developers need to view authentication statistics about the login attempts to the application. This tutorial demonstrates how the Pickup developers can setup WSO2 IS Analytics to view login attempts to the Pickup application. 
+To do this, the developers need to view authentication statistics about the login attempts to the application. This tutorial demonstrates how the Pickup developers can setup WSO2 IS Analytics to view login attempts to the Pickup application.
 
 ## Set up 
 
 1. [Download WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
 
-2. [Download WSO2 Identity Server Analytics distribution](https://wso2.com/identity-and-access-management/install/analytics/).
+2. [Download WSO2 Identity Server Analytics distribution](https://github.com/wso2/analytics-is/releases/tag/v5.8.0-rc3).
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-identity-analytics.md:15]: Why are we downloading an RC pack here?

File: en/docs/learn/configuring-identity-analytics.md [hunk 2]
```diff
@@ -6,27 +6,60 @@ Using the WSO2 Identity Server Analytics distribution, you can view and analyze
...
 
 ## Enable analytics
+Follow the steps below to configure WSO2 Identity Server Analytics.
 
-Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and enable the following event publishers to enable analytics in WSO2 Identity Server. 
+1.	Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and enable the following event publishers to enable analytics in WSO2 Identity Server.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-identity-analytics.md:20]:
```suggestion
1.	Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and enable the following event publishers to enable analytics in WSO2 Identity Server:
```

File: en/docs/learn/configuring-identity-analytics.md [hunk 3]
```diff
@@ -6,27 +6,60 @@ Using the WSO2 Identity Server Analytics distribution, you can view and analyze
...
+    enable=true
+    ```
+
+    ```toml
+    [identity_mgt.analytics_session_data_publisher] 
+    enable=true
+    ```
+
+
+2.  An HTTP connection is used to communicate between WSO2 IS and WSO2
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-identity-analytics.md:33]: Suggestion:

```
2. Import the certificate from WSO2 IS Analytics to WSO2 IS. That is, import the
    `           public certificate          ` of each keystore to the
    `           client­-truststore.jks          ` of WSO2 IS. 
    
    This example uses the default keystores and certificates.
    
    !!! info
        An HTTP connection is used to communicate between WSO2 IS and WSO2 IS Analytics. Therefore, you must add the certificate of WSO2 IS Analytics to WSO2 IS.
```

File: en/docs/learn/configuring-identity-analytics.md [hunk 4]
```diff
@@ -6,27 +6,60 @@ Using the WSO2 Identity Server Analytics distribution, you can view and analyze
...
+    IS Analytics. Therefore, you must add the certificate of WSO2 IS Analytics to WSO2 IS.
+    Follow the steps given below to import the certificate from WSO2 IS Analytics
+    to WSO2 IS. Import the
+    `           public certificate          ` of each keystore to the
+    `           client­-truststore.jks          ` of the WSO2 IS. This example uses the default keystores and
+    certificates.
+
+    1.  Navigate to the
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-identity-analytics.md:41]:
```
1.  Navigate to the
        `             <ISANALYTICS_HOME>/resources/security            `
        directory on a new terminal window and run the following
        command. 
        
        !!! info
            The default keystore password is `wso2carbon`.
```

File: en/docs/learn/configuring-identity-analytics.md [hunk 5]
```diff
@@ -6,27 +6,60 @@ Using the WSO2 Identity Server Analytics distribution, you can view and analyze
...
 
-``` toml
-[identity_mgt.analytics_login_data_publisher]
-enable=true
-```
+        ``` java
+        keytool -export -alias wso2carbon -keystore wso2carbon.jks -file sp.cer
+        ```
+
+    2.  Navigate to the
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-identity-analytics.md:50]:
```
2.  Navigate to the
        `             <IS_HOME>/repository/resources/security            `
        directory and run the following command:
```

File: en/docs/learn/configuring-identity-analytics.md (hunk 1 above)
Comment (Sachin-Mamoru) [reply en/docs/learn/configuring-identity-analytics.md:15]: For the 5.11.0 branch the given url is this. But for the master branch as you can see the above url is invalid. Therefore I have override the provided url with the 5.11.0 branch given url.

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**